tests/unit/testplan/testing/multitest/test_multitest.py

```diff
@@ -0,0 +1,281 @@
+"""Unit tests for MultiTest base functionality."""
+
+import os
+
+from testplan.common.utils import path
+from testplan.testing import multitest
+from testplan.testing.multitest import base
+from testplan.testing.multitest.entries import assertions
+from testplan.testing import filtering
+from testplan.testing import ordering
+from testplan import defaults
+from testplan import report
+from testplan.common import entity
+from testplan.testing.multitest.entries import schemas
+
+# TODO: shouldn't need to specify these...
+MTEST_DEFAULT_PARAMS = {
+    "test_filter": filtering.Filter(),
+    "test_sorter": ordering.NoopSorter(),
+    "stdout_style": defaults.STDOUT_STYLE,
+}
+
+
+def test_multitest_runpath():
+    # # No runpath specified
+    mtest = multitest.MultiTest(
+        name="Mtest", suites=[], **MTEST_DEFAULT_PARAMS
+    )
+    assert mtest.runpath is None
+    assert mtest._runpath is None
+    mtest.run()
+    assert mtest.runpath == path.default_runpath(mtest)
+    assert mtest._runpath == path.default_runpath(mtest)
+
+    # runpath in local cfg
+    custom = os.path.join("", "var", "tmp", "custom")
```

Comment:
Nitpick: This could just be /var/tmp/custom since Windows won't have this anyway.

Reply:
I think this test still runs on Windows. I don't think it particular matters whether this path actually exists or not, I think it's just testing that it can be set as an attribute. For now I think safest to leave as is.